here/here_readme.md

```diff
@@ -0,0 +1,218 @@
+
+### Traffic Patterns: Data Model 
+
+The Traffic Patterns dataset comes in 15-min and 60-min models, which have a basic wide structure of `{pattern_id, h00_00, h00_015, [...], h23_45}` where `hHH_MM` is the speed value for `pattern_id` for that time of day. 
+
+Both the 15-min and 60-min tables share the same `pattern_ids`, which can be found in the lookup reference table `here.traffic_pattern_YY_ref` (where `YY` is the year of the model). This table is of the format `{link_pvid, travel_direction, u, m, t, w, r, f, s}` where each of those letter columns contains a `pattern_id` for that combination of `link_dir` and `day of the week` starting with sUnday.
+
+`here.traffic_pattern_YY_ref_narrow`
+
+|column | type | definition |
+|-------|------|------------|
+|link_dir |text | link direction |
+|isodow |integer | ISO Day of Week|
+|pattern_id |integer | id referring to the pattern for that link_dir, day of week combination|
+
+The tables are converted to a narrow format (which means that the 15-min and 60-min time periods show up in one field (`trange`) instead of each time period taking up its own column). The 15-min and 60-min tables have the same format.
+
+`here.traffic_pattern_YY_spd_MM_narrow`
+
+|column | type | definition |
+|-------|------|------------|
+|pattern_id| integer| ID for this pattern|
+|trange| timerange| Time range for which this pattern applies|
+|pattern_speed| integer|Speed in km/hr for that pattern_id & time range|
+
+Check out this [documentation](here/here_loading.md) for more info on how the traffic patterns tables are loaded and transformed.
+
+## Routing with Traffic Data
+
+One use of historical traffic data is the ability to route a vehicle from an
+arbitrary point to another arbitrary point using traffic data **at that point
+in time**. Since our data are already in a database, this can be accomplished
+using the [`pgRouting`](http://pgrouting.org/) PostgreSQL extension. It is
+necessary to have [traffic patterns](#traffic-patterns-traffic-models) loaded
+to fill in gaps in traffic data in time.
+
+The following views prepare the HERE data for routing (code found
+[here](traffic/sql/create_here_routing.sql)):
+
+- `here.routing_nodes`: a view of all intersections derived from the `z_levels`
+  gis layer.
+- `here.routing_streets_YY_Q`: The geography of streets is provided as
+  centerlines, but traffic is provided directionally. This view creates
+  directional links for each permitted travel direction on a navigable street
+  with a `geom` drawn in the direction of travel.
+
+Its a good idea to make sure that your tables or views are from the same time period, or as close to the same time period, as possible. Due to some inconsistencies in what we receive from MTO, perfect time period matches are not always possible. For example, as of July 2022:
```

Comment:
~MTO~ **HERE**

Reply:
Where/how should we store the information on which ta tables correspond to which map versions?

Reply:
We want to add something about what ta data links to what street network... we might have to go digging in the issues to find incidences of HERE map updates...
We do know that 2019 to present = 21_1. We can investigate this more later. For now we should say that it's important to ensure that the map and ta data match. An easy way to check this is by joining the link_dirs and seeing what's left lonely and unjoined.

Reply:
Implemented